js/app.js

```diff
@@ -1 +1,126 @@
+var player1 = "X";
+var player2 = "O";
+var arrayX = [];
+var arrayO = [];
+
+
+// EVENT LISTENERS
+
+	var playerMove = document.getElementsByClassName('cell');
+	
+	Array.from(playerMove)
+		.forEach(function (click) { 
+  			click.addEventListener('click', function(e) {
+     		click.textContent = turn;
+     		switchTurn();
+     		winner();
+     		e.target.removeEventListener(e.type, arguments.callee);
+     		});
+  		})
+  	
+  
+// FUNCTIONS
+
+function switchTurn() {
+ if (turn == "X") {
+   turn = "O";
+ } 
+ else {
+   turn = "X";
+   }
+}
+
+function winner () {
+	if(cell1.textContent === player1 && cell2.textContent === player1 && cell3.textContent === player1){
+		alert("X is the winner!");
+	}
+	else if(cell4.textContent === player1 && cell5.textContent === player1 && cell6.textContent === player1){
+		alert("X is the winner!");
+	}
+	else if(cell7.textContent === player1 && cell8.textContent === player1 && cell9.textContent === player1){
+		alert("X is the winner!");
+	}
+	else if(cell1.textContent === player1 && cell4.textContent === player1 && cell7.textContent === player1){
+		alert("X is the winner!");
+	}
+	else if(cell2.textContent === player1 && cell5.textContent === player1 && cell8.textContent === player1){
+		alert("X is the winner!");
+	}
+	else if(cell3.textContent === player1 && cell6.textContent === player1 && cell9.textContent === player1){
+		alert("X is the winner!");
+	}
+	else if(cell1.textContent === player1 && cell5.textContent === player1 && cell9.textContent === player1){
+		alert("X is the winner!");
+	}
+	else if(cell7.textContent === player1 && cell5.textContent === player1 && cell3.textContent === player1){
+		alert("X is the winner!");
+	}
+
+	
+	if(cell1.textContent === player2 && cell2.textContent === player2 && cell3.textContent === player2){
+		alert("O is the winner!");
+	}
+	else if(cell4.textContent === player2 && cell5.textContent === player2 && cell6.textContent === player2){
+		alert("O is the winner!");
+	}
+	else if(cell7.textContent === player2 && cell8.textContent === player2 && cell9.textContent === player2){
+		alert("O is the winner!");
+	}
+	else if(cell2.textContent === player2 && cell4.textContent === player2 && cell7.textContent === player2){
+		alert("O is the winner!");
+	}
+	else if(cell2.textContent === player2 && cell5.textContent === player2 && cell8.textContent === player2){
+		alert("O is the winner!");
+	}
+	else if(cell3.textContent === player2 && cell6.textContent === player2 && cell9.textContent === player2){
+		alert("O is the winner!");
+	}
+	else if(cell2.textContent === player2 && cell5.textContent === player2 && cell8.textContent === player2){
+		alert("O is the winner!");
+	}
+	else if(cell7.textContent === player2 && cell5.textContent === player2 && cell3.textContent === player2){
+		alert("O is the winner!");
+	}
+	else if(cell1.textContent === player2 && cell5.textContent === player2 && cell9.textContent === player2){
+		alert("O is the winner!");
+	}
+
```

Comment:
If you wanted to alert a draw, think you can create a variable that keeps a count of the turns (adding a count++ or what not to your switchTurn function). Then after the end of the function Winner, you can have a else if statement seeing if the (count === 8) if you start from 0, which would alert a draw only if there wasn't a winner otherwise.